core/logic.py

```diff
@@ -138,8 +142,9 @@ def get_prettified_schedule(locale, schedule, selector):
 
 
 def on_message(reply: BaseReply, message_text):
-    locale = locale_manager.read_locale()
-    if message_text.startswith(locale_manager.read_prefix(locale)):
+    with open(file=loc+'.json', encoding="UTF-8") as file:
```

Comment:
Why did you remove localization class from the code and added direct file loading?